demos/serverless/deploy.js

```diff
@@ -144,6 +178,53 @@ function ensureTools() {
   spawnOrFail('npm', ['install']);
 }
 
+function createCaptureS3Buckets(bucketPrefix, regions) {
+  console.log(`Creating S3 buckets for media capture pipelines artifacts.  Bucket prefix: ${bucketPrefix} Regions:[${regions}]`);
```

Comment:
Can this not be created from the SAM template?

Reply:
Unfortunately not because the SAM template can not create resources in more than one region.